modules/templates/helper.go

```diff
@@ -795,6 +796,13 @@ func RenderCommitBody(ctx context.Context, msg, urlPrefix string, metas map[stri
 	return template.HTML(renderedMessage)
 }
 
+var codeMatcher = regexp.MustCompile("`([^`]+)`") // capture strings of the format `code-like text`
```

Comment:
Comments are usually placed on a separate line.

```suggestion
// Match text that is between back ticks. 
var codeMatcher = regexp.MustCompile("`([^`]+)`")
```